slides/CM5/CM5_Tests_Variable_Selection.Rmd

```diff
@@ -347,13 +350,12 @@ $$
 Reject Region:
 $$
 \mathbb{P}[\text{Reject} | \mathcal{H}_0 \text{ true}] \leq \alpha
+\qquad\qquad\qquad\qquad\qquad\\
```

Comment:
Same remark for this slide, see complied version [here](https://hmma201.netlify.app/hmma201/slides/cm5/cm5_tests_variable_selection#19)
If the linked slide looks ok, then maybe these formatting change are not necessary ?